umm-spec-lib/src/cmr/umm_spec/migration/version/collection.clj

```diff
@@ -740,4 +740,60 @@
   [_context collection & _]
   ;; File sizes from positive numbers to numbers, only need to migrate version. 
   (-> collection
-      (m-spec/update-version :collection "1.18.0"))) 
+      (m-spec/update-version :collection "1.18.0")))
+
+(defmethod interface/migrate-umm-version [:collection "1.18.1" "1.18.2"]
+   [_context collection & _]
+   ;; Migrating up version 1.18.1 to 1.18.2
+   ;; Add AssociatedDOIs/Type enums: IsPreviousVersionOf and IsNewVersionOf
+   ;; Add PREPRINT, INREVIEW, and SUPERSEDED enums to CollectionProgress
+   ;; Remove NOT APPLICABLE enum from CollectionProgress
+   (-> collection
+       (m-spec/update-version :collection "1.18.2")
+       ;; Change CollectionProgress to "NOT PROVIDED" if its value is "NOT APPLICABLE"
+       (as-> coll (if (= "NOT APPLICABLE" (:CollectionProgress coll))
+                    (-> coll
+                        (update :CollectionProgress (constantly "NOT PROVIDED")))
+                    coll))))
+
+(defn- migrate-associated-doi-type-down
+       [associatedDOI]
+       (if (or (= "IsPreviousVersionOf" (:Type associatedDOI))
+               (= "IsNewVersionOf" (:Type associatedDOI)))
+               (-> associatedDOI
+                   (update :Type (constantly "Related Dataset"))
+                   (remove-nil-keys))
+               associatedDOI))
+
+(defn- migrate-collection-progress-down
+       [collectionProgress]
+
+       (if (nil? collectionProgress)
+         "NOT PROVIDED"
+         (if (or (= "PREPRINT" collectionProgress)
+                 (= "INREVIEW" collectionProgress)
+                 (= "SUPERSEDED" collectionProgress))
+           "PLANNED"
+           collectionProgress)))
```

Comment:
From 1.18.2 to 1.18.1, if PREPRINT, INREVIEW,  use PLANNED instead.  for SUPERSEDED are use COMPLETED instead

Reply:
done